ngraph/frontend/paddlepaddle/src/op/conv2d.cpp

```diff
@@ -28,20 +18,46 @@ namespace ngraph
                 NamedOutputs conv2d(const NodeContext& node)
                 {
                     auto data = node.get_ng_input("Input");
-                    auto filter = node.get_ng_input("Filter");
-                    // TODO: resolve padding according to spec
-                    auto strides = node.get_attribute<std::vector<int32_t>>("strides");
-                    auto paddings = node.get_attribute<std::vector<int32_t>>("paddings");
-                    auto dilations = node.get_attribute<std::vector<int32_t>>("dilations");
-                    return node.default_single_output_mapping(
-                        {std::make_shared<ngraph::opset6::Convolution>(
-                            data,
-                            filter,
-                            ngraph::Strides(strides.begin(), strides.end()),
-                            ngraph::CoordinateDiff(paddings.begin(), paddings.end()),
-                            ngraph::CoordinateDiff(paddings.begin(), paddings.end()),
-                            ngraph::Strides(dilations.begin(), dilations.end()))},
-                        {"Output"});
+                    auto filters = node.get_ng_input("Filter");
```

Comment:
Can this be combined with conv2d_transpose? The only difference now is data type (GroupConvolutionBackpropData-GroupConvolution and ConvolutionBackpropData-Convolution)
So we can probably use templates for this, like
`template<T1, T2> NamedOutputs conv2d_base(node) {....}`
And then define
```
NamedOutputs conv2d(node) {
   return conv2d_base<GroupConvolution, Convolution>(node);
}

NamedOutputs conv2d_transpose(node) {
   return conv2d_base<GroupConvolutionBackPropData, ConvolutionBackPropData>(node);
}

```